public/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
+    <link rel="stylesheet" href="/default/css/default.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-    <p>
-       Open browser console and play with <code>hoodie.account</code>.
-    </p>
+
+    <div data-hide="false" class="generic-loader-wrap"></div>
+
+    <div data-hide="true" class="input-forms">
```

Comment:
These are all sections of content, so sections seems to make sense.
I try not to use `div` wherever possible as `div` is not very semantic HTML.

http://html5doctor.com/lets-talk-about-semantics/ might explain better.

a better question might be, why don't you think we should use them? :)